clients/admin-ui/src/features/connection-type/types.ts

```diff
@@ -14,7 +10,7 @@ export type AllConnectionTypesResponse = {
 };
 
 export type ConnectionOption = {
-  identifier: ConnectionType | SaasType;
+  identifier: string;
```

Comment:
Is this loss of specificity because the API doesn't type the value correctly, or is there a real use case for this being a freeform string?

Reply:
Related: a while back [I added](https://github.com/ethyca/fides/pull/863) OpenAPI type generation. Running that that now modifies ~200 files becuase it pulls in Ops models now. Instead of grabbing those in a big bang, we could replace the manually defined ones as we encounter them. For example, this would become `src/types/api/models/ConnectionSystemTypeMap.ts`